docs/cubeviz/displaycubes.rst

```diff
@@ -2,7 +2,23 @@
 Displaying Cubes
 ****************
 
-Words go here
+The Cubeviz layout includes three image viewers (at the top of the app)
+and one spectrum viewer (at the bottom of the app), which it attempts to 
+populate automatically when the first dataset is loaded. By default, cubeviz
+attempts to parse and display the flux in the top left viewer, the uncertainty
+in the top middle viewer, and the mask into the top right viewer. The spectrum
+viewer is populated by default by collapsing the spatial axes using the `max`
+function. 
+
```

Comment:
It would be useful to specifically mention what cubeviz looks for to determine the flux, uncertainty, and mask extensions.

Reply:
I added some more information on that in the most recent commit, thanks for the suggestion.